reference/checkouts.v3.yml

```diff
@@ -1293,9 +1211,7 @@ definitions:
                         description: The URL for the combined downloads page.
                         type: string
                       download_size:
-                        description: >-
-                          Combined download size, in human-readable style. E.g.:
-                          `30MB`.
+                        description: 'Combined download size, in human-readable style. For example, `30MB`.'
                         type: string
                     required:
                       - quantity
```

Comment:
`digital_items` element is missing the following properties:` is_mutable`, `is_require_shipping`, `parent_id`.